### PR TITLE
Feat/add res req size metric

### LINKED ISSUE #32: Add response size metrics
I think it might be useful to add a response size metric in the set of stats collected. WDYT?

### PR DESCRIPTION
**Pull request by simonecorsi**
In this PR I reopen discussion on request/resposne size metrics, added a first draft implementation, PTAL

closes #32

### CODE REVIEW

**Comment by antoniomuso** [comment]
Waiting for: https://github.com/immobiliare/fastify-metrics/pull/133

File: lib/hooks.js [hunk 1]
```diff
@@ -15,6 +15,22 @@ exports.onResponse = function (request, reply, next) {
     next();
 };
 
+exports.onRequestSize = function (request, reply, next) {
+    const contentLength = request.headers?.['content-length'];
```
Comment (antoniomuso) [inline lib/hooks.js:19]: We need to remove this, because not supported in node 12

Comment (simonecorsi) [reply lib/hooks.js]: ThatThis is already been done :+1:

**Review by dnlup — APPROVED** [review]
lgtm, thanks. How do we want to proceed for the readme update? do we want to merge this and add the docs later in #130 ? I am ok with it

**Comment by dnlup** [comment]
I'll merge it, then add the docs in #130

**Pull request merged**